Don't include altivec.h as this break the build on powerpc64.

Index: gegl/opencl/cl_platform.h
--- gegl/opencl/cl_platform.h.orig
+++ gegl/opencl/cl_platform.h
@@ -270,7 +270,6 @@ typedef unsigned int cl_GLenum;
 
 /* Define basic vector types */
 #if defined( __VEC__ )
-   #include <altivec.h>   /* may be omitted depending on compiler. AltiVec spec provides no way to detect whether the header is required. */
    typedef vector unsigned char     __cl_uchar16;
    typedef vector signed char       __cl_char16;
    typedef vector unsigned short    __cl_ushort8;
